Explorer/Assets/DCL/AvatarRendering/AvatarShape/Systems/AvatarLoaderSystem.cs

```diff
@@ -96,7 +96,7 @@ private void UpdateAvatarFromProfile(ref Profile profile, ref AvatarShapeCompone
 
         private Promise CreateWearablePromise(Profile profile, PartitionComponent partition) =>
             Promise.Create(World,
-                WearableComponentsUtils.CreateGetWearablesByPointersIntention(profile.Avatar.BodyShape, profile.Avatar.SharedWearables, profile.Avatar.ForceRender),
+                WearableComponentsUtils.CreateGetWearablesByPointersIntention(profile.Avatar.BodyShape, profile.Avatar.UniqueWearables, profile.Avatar.ForceRender),
```

Comment:
Is crucial to be able to index the wearables by the extended urn. If we ask for `shared`, im not going to be able to add the unique wearable in the catalog, thus it will fail when requesting the profile wearables from the backpack or anywhere else